expansions/jesters_gambit/README.md

```diff
@@ -0,0 +1,17 @@
+# Jester's Gambit
+
+This expansion introduces an additional way for Lads to gain extra flips on their turn by performing entertaining tricks.
+
+## Required Materials
+
+No additional materials required
+
+## Additional Rules
+
+During the Flip phase before flipping the hammer, if the active player is a Lad, they may call "Jester" before flipping the hammer.
+
+If they call "Jester", the Lad becomes a Jester for the remainder of their turn. The Jester must perform a trick with the hammer that is incorporated into their Flip phase (e.g.: flipping the hammer behind their back, or under their leg, etc.). The Jester must describe the trick they plan to perform. If this trick does not sound adequately exciting as outlined, the other players may vote by majority to boo the Jester. The Jester must take one (1) drink and has the opportunity to present another trick idea or to end their turn. This process continues until either their turn ends or the players agree that a trick sounds entertaining. The Jester then proceeds to make their flip.
```

Comment:
@austenlux note one small tweak I made here.

I made it so the Jester has to pitch what they are going to do and people can veto it. I'm thinking otherwise people can just say they are doing a flip and do something boring/safe